app/App.tsx

```diff
@@ -1,8 +1,8 @@
+// eslint-disable-next-line camelcase
+import { Poppins_400Regular, useFonts } from '@expo-google-fonts/poppins';
```

Comment:
does this mean, that we no longer need the font in the assets?

Reply:
Good catch! 🎾 I'll have a look later at whether this can also be removed 💯

Reply:
I created issue #46 for it, so that it no longer blocks this PR